guides/v2.1/cloud/docker/docker-development.md

```diff
@@ -33,22 +33,21 @@ The [Magento Cloud Docker repository](https://github.com/magento/magento-cloud-d
 
 The web container works with the [PHP-FPM](https://php-fpm.org) to serve PHP code, the **DB** image for the local database, and the **Varnish** image to send requests and cache the results.
 
-### CLI container
+### CLI containers
 
-The CLI container is based on a PHP-CLI image that provides `magento-cloud` and `{{site.data.var.ct}}` commands to perform file system operations. The CLI container depends on the **DB** image for the local database and the **Redis** image.
+There are few CLI containers based on a PHP-CLI image that provides `magento-cloud` and `{{site.data.var.ct}}` commands to perform file system operations.
```

Comment:
So, this indicates there are multiple containers, but down below it says these things "extend" a container (singular). Maybe we need to be more specific about the containers, and I could add some examples.

Reply:
Yeah, it's better to change it to `extends PHP-CLI image`